mkdocs.yml

```diff
@@ -5,14 +5,17 @@ docs_dir: 'markdown-docs'
 google_analytics: ['UA-991100-5', 'search.asf.alaska.edu']
 
 theme:
-    name: material
+    name: asf-theme
     custom_dir: asf-theme
     favicon: assets/images/favicon.ico
     logo: assets/images/ASFLogo-Blue2.png
```

Comment:
Looks good Andy! 

You shouldn't need these three lines:

```
    custom_dir: asf-theme
    favicon: assets/images/favicon.ico
    logo: assets/images/ASFLogo-Blue2.png
```

As they are included with the theme. You may want to turn off the site logo if you only want the ASF logo:

https://github.com/ASFHyP3/mkdocs-asf-theme#logos